### PR TITLE
Feature/ Implement Cards component

### PR DESCRIPTION
**Pull request by JennyDoJD**
Result: 
image

image

Ticket: [Implement Cards common component](https://trello.com/c/z9Kc4MyW/26-implement-cards-common-component)

### CODE REVIEW

File: practice/react-practice-one/src/components/Cards/Card.jsx [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+  description: PropTypes.shape({
+    price: PropTypes.string,
+    quantity: PropTypes.string,
+  }),
+  onEditClick: PropTypes.func,
+  onAddClick: PropTypes.func,
+  text: PropTypes.string.isRequired,
+};
+
+export default Card;
```
Comment (ghost) [inline practice/react-practice-one/src/components/Cards/Card.jsx]: the name should be `ProductCard`

Comment (JennyDoJD) [reply practice/react-practice-one/src/components/Cards/Card.jsx]: I fixed it

File: practice/react-practice-one/src/components/Cards/ProductCard.jsx [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+          onClick={onAddClick}
+        >
+          <AddIcon />
+          <Text variant="secondary" className="text-title-sm">
+            {text}
+          </Text>
+        </a>
+      )}
+      {type === CARD_TYPES.PRODUCT && (
+        <div className="card card-product">
```
Comment (ghost) [inline practice/react-practice-one/src/components/Cards/ProductCard.jsx]: @JennyDoJD should we need update class name as well?

Comment (JennyDoJD) [reply practice/react-practice-one/src/components/Cards/ProductCard.jsx]: I fixed it

File: practice/react-practice-one/src/components/Cards/AddProductCard.stories.jsx [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+      action: 'add-clicked',
+      description: 'Callback function for add button click',
+    },
+  },
+};
+
+export default meta;
+
+export const CardAddProduct = (args) => <ProductCard {...args} />;
+CardAddProduct.args = {
```
Comment (ghost) [inline practice/react-practice-one/src/components/Cards/AddProductCard.stories.jsx]: CardAddProduct => AddProductCard

File: practice/react-practice-one/src/components/Cards/ProductCard.stories.jsx [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+    onDeleteClick: {
+      action: 'delete-clicked',
+      description: 'Callback function for delete icon click',
+    },
+  },
+};
+
+export default meta;
+
+export const CardProduct = (args) => <ProductCard {...args} />;
```
Comment (ghost) [inline practice/react-practice-one/src/components/Cards/ProductCard.stories.jsx]: CardProduct => ProductCard

Comment (ghost) [reply practice/react-practice-one/src/components/Cards/ProductCard.stories.jsx]: @JennyDoJD please double-check and pay more to attend before requesting review

File: practice/react-practice-one/src/components/Cards/AddProductCard.stories.jsx (hunk 3 above)
Comment (JennyDoJD) [reply practice/react-practice-one/src/components/Cards/AddProductCard.stories.jsx]: I done

File: practice/react-practice-one/src/components/Cards/ProductCard.stories.jsx (hunk 4 above)
Comment (JennyDoJD) [reply practice/react-practice-one/src/components/Cards/ProductCard.stories.jsx]: I done

**Review by ghost — APPROVED** [review]

**Pull request merged**